### DISCUSSION TITLE
New edition with type hinting?

### DISCUSSION THREAD

**Discussion by staticdev** [pycabook/pycabook#33] [Ideas]
This is the new hot in Python, many PEPs regading this subject, even in Python 3.9. If you intend to add that to your book I offer my help to get it done =)

**Comment by lgiordani** [comment]
Hey @staticdev sorry for the late reply, I didn't meant o ignore you. I'm actually writing the second edition of the book and I am glad to hear suggestions on that side. Feel free to post them here, we can discuss them and see if they are interesting and if it's possible to fit them into the release schedule. Thanks!

**Comment by staticdev** [comment]
@lgiordani I am also refactoring some projects to a more clean-ish achitecture and you will see that I already have some similar code with type hints, check this:
https://github.com/staticdev/git-portfolio/blob/master/src/git_portfolio/response_objects.py
https://github.com/staticdev/git-portfolio/blob/master/tests/test_response_objects.py

**Comment by lgiordani** [comment]
Thanks! I will have a look
